driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideEncryptionTest.java

```diff
@@ -0,0 +1,324 @@
+/*
```

Comment:
This isn't really new, but git seems to think it is.  It's just the old ClientSideEncryptionTest renamed